eventmesh-protocol-plugin/eventmesh-protocol-meshmessage/src/main/java/org/apache/eventmesh/protocol/meshmessage/resolver/grpc/GrpcMessageProtocolResolver.java

```diff
@@ -41,7 +40,47 @@
 
 public class GrpcMessageProtocolResolver {
 
+    private static CloudEvent getBuilderCloudEvent(SendMessageRequestBody sendMessageRequestBody, String protocolType, String protocolDesc, 
```

Comment:
I don't understand why the parameter sendMessageRequestBody is passed here, I think it can be simplified as follows:
```
private static CloudEvent getBuilderCloudEvent(RequestHeader requestHeader, String content, CloudEventBuilder cloudEventBuilder,
                                                   String seqNum, String topic, String uniqueId, String producerGroup, String ttl,
                                                   Map<String, String> propertiesMap, String tag) {
    final CloudEvent event;
    cloudEventBuilder = cloudEventBuilder.withId(seqNum)
        .withSubject(topic)
        .withType("eventmeshmessage")
        .withSource(URI.create("/"))
        .withData(content.getBytes(StandardCharsets.UTF_8))
        .withExtension(ProtocolKey.ENV, requestHeader.getEnv())
        .withExtension(ProtocolKey.IDC, requestHeader.getIdc())
        .withExtension(ProtocolKey.IP, requestHeader.getIp())
        .withExtension(ProtocolKey.PID, requestHeader.getPid())
        .withExtension(ProtocolKey.SYS, requestHeader.getSys())
        .withExtension(ProtocolKey.USERNAME, requestHeader.getUsername())
        .withExtension(ProtocolKey.PASSWD, requestHeader.getPassword())
        .withExtension(ProtocolKey.LANGUAGE, requestHeader.getLanguage())
        .withExtension(ProtocolKey.PROTOCOL_TYPE, requestHeader.getProtocolType())
        .withExtension(ProtocolKey.PROTOCOL_DESC, requestHeader.getProtocolDesc())
        .withExtension(ProtocolKey.PROTOCOL_VERSION, requestHeader.getProtocolVersion())
        .withExtension(ProtocolKey.SEQ_NUM, seqNum)
        .withExtension(ProtocolKey.UNIQUE_ID, uniqueId)
        .withExtension(ProtocolKey.PRODUCERGROUP, producerGroup)
        .withExtension(ProtocolKey.TTL, ttl);

      for (Map.Entry<String, String> entry : propertiesMap.entrySet()) {
        cloudEventBuilder.withExtension(entry.getKey(), entry.getValue());
    }
    if (StringUtils.isNotEmpty(tag)) {
        cloudEventBuilder = cloudEventBuilder.withExtension(ProtocolKey.TAG, tag);
    }
    event = cloudEventBuilder.build();
    return event;
}
```
What do you think?